tutorials/quickstarts/validated_linux_requirements.txt

```diff
@@ -0,0 +1,119 @@
+adal<=1.2.6
```

Comment:
this is a scary list of pinned dependencies

Reply:
Deleted this file, not needed.